dynamics-nav/Accessing-Microsoft-Dynamics-NAV-Tables-with-External-Tools.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Accessing Microsoft Dynamics NAV Tables with External Tools"
+description: "This article explains accessing data in Microsoft Dynamics NAV 2018 tables with external tools. Additional references are provided."
```

Comment:
The second sentence doesn't really add anything.  Feel free to keep this fix "as is" though.